newsletter-march-2022.rst

```diff
@@ -0,0 +1,63 @@
+.. post:: Mar 10, 2022
+   :tags: newsletter, python
+   :author: Eric Holscher
+   :location: BND
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - March 2022
+=====================================
+
+It's been pretty quiet on the company front in February,
+with nothing much to report.
+**We're actively working on our latest job description,
+which will be a product-focused Python development position.**
+f you're interested, please `let us know`_.
```

Comment:
```suggestion
If you're interested, please `let us know`_.
```

Reply:
I'm not sure this will bring us good candidates, tho. I mean, without a public job description; but I'm fine keeping it.